data/poke/src/main/java/org/sopt/official/data/poke/dto/response/PokeFriendOfFriendListResult.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * MIT License
+ * Copyright 2023 SOPT - Shout Our Passion Together
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.sopt.official.data.poke.dto.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.sopt.official.domain.poke.entity.PokeFriendOfFriendList
+
+@Serializable
+data class PokeFriendOfFriendListResult(
+    @SerialName("friendId")
+    val friendId: Long,
+    @SerialName("playgroundId")
+    val playgroundId: Int,
+    @SerialName("friendName")
+    val friendName: String,
+    @SerialName("friendProfileImage")
+    val friendProfileImage: String,
+    @SerialName("friendList")
+    val friendList: List<PokeUserResult>
+) {
+    fun toEntity(): PokeFriendOfFriendList = PokeFriendOfFriendList(
```

Comment:
```suggestion
    fun toEntity() = PokeFriendOfFriendList(
```

타입이 너무 길어서 불필요한 타입선언은 지워도 될 것 같습니다. 
이렇게만해도 리턴타입 확실하게 보일 것 같아요.